doc/nrf/protocols/matter/end_product/test_event_triggers.rst

```diff
@@ -0,0 +1,229 @@
+
+.. _matter_test_event_triggers_default_test_event_triggers:
+
+Default test event triggers
+===========================
+
+You can use the pre-defined common test event triggers in your application.
+The default test triggers are enabled by default.
+To disable them, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_TEST_EVENT_TRIGGERS_REGISTER_DEFAULTS` Kconfig option to ``n``.
+
+The following table lists the available triggers and their activation codes:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+   * - Name
+     - Requirements
+     - Description
+     - Activation code [range]
+     - Value description
+   * - Factory reset
+     - None
+     - Perform a factory reset of the device with a delay.
+     - `0xF000000000000000` - `0xF00000000000FFFF`
+     - The range of `0x0000` - `0xFFFF` represents the delay in ms to wait until the factory reset occurs.
+       The maximum time delay is UINT32_MAX ms.
+       The value is provided in HEX format.
+   * - Reboot
+     - None
+     - Reboot the device.
+     - `0xF000000000000000` - `0xF00000000000FFFF`
+     - The range of `0x0000` - `0xFFFF` represents the delay in ms to wait until the reboot occurs.
+       The maximum time delay is UINT32_MAX ms.
+       The value is provided in HEX format.
+   * - OTA query
+     - :kconfig:option:`CONFIG_CHIP_OTA_REQUESTOR` = ``y``
+     - Trigger an OTA firmware update.
+     - `0x0100000000000100` - `0x01000000000001FF`
+     - The range of `0x00` - `0xFF` is the fabric index value.
+       The maximum fabric index value depends on the current device's settings.
+   * - Door lock jammed
+     - :kconfig:option:`CONFIG_CHIP_DEVICE_PRODUCT_ID` = ``32774``
+     - Simulate the jammed lock state.
+     - `0x3277400000000000`
+     - This activation code does not contain any value.
+
+.. _matter_test_event_triggers_setting_enable_key:
+
+Setting the enable key
+======================
+
+The enable key can be provided utilizing the factory data or directly in the application code.
+
+Using factory data
+------------------
+
+You can set the enable key to a specific value only if the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA` Kconfig option is set to ``y``.
+Otherwise, the default value ``00112233445566778899AABBCCDDEEFF`` will be used.
+
+To specify the enable key through the build system, enable the :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` Kconfig option by setting it to ``y``.
+Then, set the :kconfig:option:`CONFIG_CHIP_DEVICE_ENABLE_KEY` Kconfig option to a 32-byte hexadecimal string value.
+
+If :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` is set to ``n``, you can use the Python script to generate the factory data set with the specific enable key value.
+To learn how to do this, read the :doc:`matter:nrfconnect_factory_data_configuration` guide in the Matter documentation.
```

Comment:
```suggestion
If :kconfig:option:`CONFIG_CHIP_FACTORY_DATA_BUILD` is set to ``n``, you can follow the :doc:`matter:nrfconnect_factory_data_configuration` guide in the Matter documentation to generate the factory data set with the specific enable key value.
```
Since we're not introducing the Python script on this page, I think this is a better wording.